src/components/LoginPage/Slide4.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import slideImg1 from 'components/LoginPage/images/slide-4_img-1.svg';
+import slideImg2 from 'components/LoginPage/images/slide-4_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 10%;
+  text-align: center;
+  position: relative;
+
+<<<<<<< HEAD
+=======
+    .shadow {
+    -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    }
+>>>>>>> 0dfe1e73b0b826d4e2cba58197e330290b9fcca4
+    .img1 {
+        position: absolute;
+        width: 60%;
+        right: 0%;
+        top: 3%;
+        z-index: 1;
+    }
+    .img2 {
+        width: 100%;
+        margin-left: -24%;
+<<<<<<< HEAD
+        -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+        filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+=======
+>>>>>>> 0dfe1e73b0b826d4e2cba58197e330290b9fcca4
+    }
+`;
+const Figure = styled.figure`
+  position: relative;
+  width: 74%;
+  margin: auto;
+  height: auto;
+`;
+const Img = styled.img`
+  width: 100%;
+  height: auto;
+`;
+const Caption = styled.svg`
```

Comment:
`Caption` is declared in every Slide file. Since it's a component that is shared between all the slides, might make sense to move it out to a separate file and import in every Slide file.

We could create a file which would export:
- `Caption`
-  `CaptionSvgTextBox`
- `CaptionSvgText`
- `Figure`

Reply:
For example we could call this file `slideComponents.jsx`.

Reply:
Done in commit [37c53ee](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/37c53ee2dddeca76a75a008408c6a511c2e6412c)